docs/intro.asciidoc

```diff
@@ -0,0 +1,54 @@
+////
+Goal of this doc:
+Provide all the information a user needs to determine if the product is a good enough fit for their use case to merit further exploration
+
+Assumptions we're comfortable making about the reader:
+* They are familiar with Elastic
+* They are familiar with OpenTelemetry
+////
+
+[[intro]]
+== Elastic Distribution for OpenTelemetry Java
+
+++++
+<titleabbrev>Introduction</titleabbrev>
+++++
+
+:language: Java
+:language_lc: java
+:distro_name: Elastic Distribution for OpenTelemetry {language}
+
+include::release-status.asciidoc[]
+
+////
+✅ Intro
+////
+The {distro_name} ("the distro") is a {language} package that provides:
+
+* An easy way to instrument your application with OpenTelemetry.
+* Configuration defaults for best usage.
+
+////
+✅ What is it?
+✅ Why use it?
+////
+A _distribution_ is a wrapper around an upstream OpenTelemetry repository with some customizations. The {distro_name} is a wrapper around the https://opentelemetry.io/docs/languages/{language_lc}[OpenTelemetry SDK for {language}] and includes the following customizations:
+
+* Uses Elastic-specific processors that ensure optimal compatibility when exporting OpenTelemetry signal data to an Elastic backend like Elastic APM server.
```

Comment:
I wonder if it makes more sense to say "the Elastic Stack" instead of "Elastic APM Server":

> Uses Elastic-specific processors that ensure optimal compatibility when exporting OpenTelemetry signal data to the Elastic Stack.

If we do decide to stick with APM Server, I don't think there's another "Elastic backend" like it, so I'd change this to:

> Uses Elastic-specific processors that ensure optimal compatibility when exporting OpenTelemetry signal data the Elastic APM server.

Hmm, although that doesn't specify what APM Server is.

I'd be curious to get @simitt or @axw's take on this. They're more up-to-date on Elastic APM terminology than I am.

Reply:
I just noticed (and like) that elsewhere in these docs we use this phrasing:

> This guide shows you how to use the Elastic Distribution for OpenTelemetry Java ("the distro") to instrument your Java application and send OpenTelemetry data to an **Elastic Observability deployment**.

Reply:
Yeah I _think_ what matters here is really that the data is destined for the Elastic Observability solution, rather than being ingested through APM Server. The way breakdown metrics need to be fixed up is specific to how APM Server maps attributes to labels, but otherwise it doesn't seem like there's a hard dependency on the ingest path.

"Elastic Observability deployment" sounds good to me.